bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -52,7 +52,7 @@ spec:
           }
 
           # Check to see if ovs is provided by the node:
-          if [[ -L '/host/etc/systemd/system/network-online.target.wants/ovs-configuration.service' ]]; then
+          if ! cat /host/etc/*release* | grep -q OPENSHIFT_VERSION=\"4\.5\"; then
```

Comment:
everyone likes the `cat | grep`

quoting doesn't look right either, in the `\.`  the `\` will be stripped by the shell first and not escaped in the grep regex.

```bash
# echo 'OPENSHIFT_VERSION="4 5"' | grep  OPENSHIFT_VERSION=\"4\.5\"
OPENSHIFT_VERSION="4 5"
```
probably best to use single quotes and let grep read all the files.

```bash
grep -q -e 'OPENSHIFT_VERSION="4\.5"' -- /host/etc/*release* 
```

Reply:
true, but I think "4 5" isn't really a valid input here. The version will either be 4.5 or 4.6 so it works OK.

Reply:
Even if it works it's bad to have code that looks like it means something other than it does.

But anyway, this will fail on RHEL 7 nodes won't it?

Reply:
Weren't we going to check for `/host/usr/local/bin/ovs-configuration.sh` ?